packages/front-end/components/Features/RevisionLog.tsx

```diff
@@ -27,24 +37,48 @@ function RevisionLogRow({ log, first }: { log: RevisionLog; first: boolean }) {
   } catch (e) {
     // Ignore
   }
-
+  let comment: string | undefined;
+  try {
+    comment = JSON.parse(log.value)?.comment;
+  } catch (e) {
+    // Ignore
+  }
+  const openContent = () => {
+    if (commentsOnly && !!comment) {
+      return <div>{comment}</div>;
+    } else {
+      return <Code language="json" code={value} />;
+    }
+  };
+  const openClickClassNames = clsx("mb-2", {
+    "cursor-pointer": !commentsOnly,
+  });
   return (
     <div className={`appbox p-2 mb-0 ${first ? "" : "mt-3"}`}>
       <div
-        className="mb-2 d-flex cursor-pointer"
+        className={openClickClassNames}
         onClick={(e) => {
           e.preventDefault();
-          setOpen(!open);
+          if (!commentsOnly) {
+            setOpen(!open);
+          }
         }}
       >
         <h3 className="mb-0">
-          {log.action} {log.subject}
+          <>
+            {log.action} {log.subject}
+          </>
         </h3>
-        <div className="ml-auto">
-          {open ? <FaAngleDown /> : <FaAngleRight />}
-        </div>
+        {commentsOnly ? (
+          <div>{openContent()}</div>
```

Comment:
This is causing it to show the full JSON details for all revision changes when requesting a review, which takes up a ton of vertical space